src/util/net.py

```diff
@@ -55,10 +55,14 @@ def prepare_search(cast_config):
 
 
 def load_user_id_whitelist(fw):
-	"""
-	Load user IDs from a FW-group-defined whitelist.
-	"""
+	"""Load user IDs from a FW-group-defined whitelist.
+
+	Args:
+		fw (flywheel.Client): Flywheel client instance.
```

Comment:
nice doc, bro. I hated going through this thing the first time under pressure and trying to figure all the objects

Reply:
Thanks,

This is my Jam.

Expect a future PR dedicated to updating all of the Docstrings.